install/components.csv

```diff
@@ -123,6 +123,9 @@ Nanometrics,Titan TACCL-BH1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Ti
 Nanometrics,Titan TACCL-N1,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
+Nanometrics,Titan TACCL-PH2,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan
+Nanometrics,Titan TACCL-PH2,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan
+Nanometrics,Titan TACCL-PH2,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan
 Nanometrics,Trillium 120QA,Broadband Seismometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Trillium-120QA
```

Comment:
One to double check (I'll have a look as well) but are the `sensor_Nanometrics-Inc_Titan` and `sensor_Nanometrics-Inc_Titan-TACCL-N1` the same response? We could looks to do some consolidation if so.

cc @ozym

Reply:
There's six resp files for Titan sensors, and they are all identical. The versions named sensor_Nanometrics_Titan do not appear to be used anywhere that I can see either
```
sensor_Nanometrics-Inc_Titan-TACCL-BH1.xml  
sensor_Nanometrics-Inc_Titan-TACCL-N1.xml  
sensor_Nanometrics-Inc_Titan.xml  
sensor_Nanometrics_Titan-TACCL-BH1.xml  
sensor_Nanometrics_Titan-TACCL-N1.xml  
sensor_Nanometrics_Titan.xml
```